Source/WebInspectorUI/UserInterface/Views/ContextMenuUtilities.js

```diff
@@ -175,6 +175,10 @@ WI.appendContextMenuItemsForSourceCode = function(contextMenu, sourceCodeOrLocat
 
     if (sourceCode instanceof WI.Resource && !sourceCode.localResourceOverride) {
         if (sourceCode.urlComponents.scheme !== "data") {
+            contextMenu.appendItem(WI.UIString("Copy as fetch"), () => {
```

Comment:
Nit: I could see this string being confusing in localization, since the `fetch` part of it should not be translated (given it refers to an API name). This feels more ambiguous than `Copy as cURL` below, so it might be worth a localization comment to make that clearer.